src/corehost/packaging/dir.props

```diff
@@ -23,7 +23,7 @@
 
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23712\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23931\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
I don't have this line being updated in my UpdateDependencies script.  I will need to fix that...